src/main/java/seedu/address/model/person/Schedule.java

```diff
@@ -143,6 +143,12 @@ public boolean remove(EventTime eventTime) {
         return schedule.remove(eventTime);
     }
 
+    /**
+     * Clears schedule completely.
+     */
+    public void clearSchedule() {
+        schedule.clear();
```

Comment:
The first and last element of the treeset blocks off the non-working hours: 0000 - 0900, 1800 - 2350. These eventimes shouldn't be cleared.

Reply:
done.

Reply:
its removed

Reply:
deleted resolved